packages/api/src/configuration/system-defaults.ts

```diff
@@ -0,0 +1,25 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+// Below is the location of the managed defaults file on different OS'
+// this will be the Managed by profile which will show some "suggested defaults" by
+// the system administrator.
+export const SYSTEM_DEFAULTS_FILE_MAC = '/Library/Application Support/com.podman.desktop/default-settings.json';
+export const SYSTEM_DEFAULTS_FILE_WINDOWS_DIR = 'PodmanDesktop';
+export const SYSTEM_DEFAULTS_FILE_WINDOWS_FILE = 'default-settings.json';
+export const SYSTEM_DEFAULTS_FILE_LINUX = '/usr/share/podman-desktop/default-settings.json';
```

Comment:
not sure why we need constants for the paths in the API ?
Is there other part than main that needs these constants ?

I was expecting it was an "implementation details" of the implementation

Reply:
was using it for GUI testing, but we can add that in a future implementation / move the constants file if need be, i'll move the file